src/app/core/header/navbar/login-dialog/login-dialog.component.scss

```diff
@@ -1,6 +1,39 @@
-.mat-dialog-content {
```

Comment:
if you're using ViewEncapsulation. None wrap your styles into component selector in order to avoid side effects